src/GitHub.Api/Git/IRepository.cs

```diff
@@ -58,8 +58,8 @@ interface IRepository : IEquatable<IRepository>
         string CurrentBranchName { get; }
 
         event Action<GitStatus> OnStatusUpdated;
-        event Action<string> OnActiveBranchChanged;
-        event Action<string> OnActiveRemoteChanged;
+        event Action<string> OnCurrentBranchChanged;
+        event Action<string> OnCurrentRemoteChanged;
```

Comment:
We use the words Active/Current interchangeably, I'm going to setting on Current.